src/python_testing/TestDefaultWarnings.py

```diff
@@ -0,0 +1,265 @@
+#
+#    Copyright (c) 2025 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+from typing import Callable
+
+import chip.clusters as Clusters
+import TC_DefaultWarnings
+from chip.testing.matter_testing import MatterBaseTest, default_matter_test_main
+from mobly import asserts
+
+
+class TestDefaultChecker(MatterBaseTest):
+    def setup_test(self):
+        self.test = TC_DefaultWarnings.DefaultChecker()
+        self.test.mark_current_step_skipped = self.mark_current_step_skipped
+        self.skipped = 0
+        super().setup_test()
+
+    def mark_current_step_skipped(self):
+        self.skipped += 1
```

Comment:
Is this not present in the matter testing? Or is this for a custom skippage?

Reply:
It is - I'm overriding it here because this is a test of the test. Added a comment. Also added a comment at the top indicating that this is a meta-test.